app/services/message_employee_matcher.rb

```diff
@@ -14,4 +14,15 @@ def run
   def day_count
     message.days_after_start
   end
+
+  def match_time(time_zone)
+    employee_current_hour = Time.current.in_time_zone(time_zone).hour
+    employee_current_minute = Time.current.in_time_zone(time_zone).min
+
+    employee_current_hour == message.time_of_day.hour && employee_current_minute == message.time_of_day.min
+  end
+
+  def message_already_sent?(employee)
```

Comment:
this is awesome - do you think we should add a spec for this service class that tests the logic in this method? (not directly, since it's private, but a test where the setup includes an already sent message)